### PR TITLE
Create FreeDM manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
This packages the Freedoom deathmatch IWAD.

I tested this locally:

`flatpak run io.github.FreeDM -host 2 -warp 01 -skill 4`
`flatpak run io.github.FreeDM -join 127.0.0.1`

### CODE REVIEW

File: io.github.FreeDM.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+			"buildsystem": "cmake",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/coelckers/gzdoom/archive/g3.2.5.tar.gz",
+					"sha256": "e9cf0aa5b7ee0b165532dee55e4965f6aabf1acadb79f7372f8e362540206748"
+				},
+				{
+					"type": "file",
+					"url": "https://github.com/coelckers/gzdoom/raw/master/soundfont/gzdoom.sf2",
```
Comment (Alexander-Wilms) [inline io.github.FreeDM.json]: This soundfont is currently not being installed when building GZdoom from source. Upstream feature request: https://forum.zdoom.org/viewtopic.php?f=15&t=59182&p=1036858#p1036858

**Comment by nedrichards** [comment]
bot, build io.github.FreeDM

**Comment by nedrichards** [comment]
We've normally used com.github rather than io.github for apps hosted there. I have no strong opinion, but wanted to bring it up.

**Comment by Alexander-Wilms** [comment]
I used the .io TLD, since they use https://freedoom.github.io/ as their homepage.

File: io.github.FreeDM.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+		"--socket=x11",
+		"--share=ipc",
+		"--share=network",
+		"--socket=pulseaudio",
+		"--env=DOOMWADDIR=/app/share/games/doom"
+	],
+	"modules": [
+		{
+			"name": "fluidsynth",
+			"buildsystem": "cmake",
```
Comment (TingPing) [inline io.github.FreeDM.json]: `cmake-ninja`

**Comment by TingPing** [comment]
bot, build io.github.FreeDM

**Comment by Alexander-Wilms** [comment]
All checks passed.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.github.FreeDM

**Comment by Alexander-Wilms** [comment]
Thanks. I accidentally declined the invitation, could you please resend it?

**Pull request closed without merging**